src/strategies/local-strategy.mjs

```
@@ -0,0 +1,41 @@
+// noinspection JSUnusedGlobalSymbols
+
+import passport from "passport";
+import { Strategy } from "passport-local";
+import { mockUsers } from "../utils/constants.mjs";
+
+passport.serializeUser((user, done) => {
+    console.log(`Inside serialize user`);
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)